src/App.js

```diff
@@ -4,18 +4,23 @@ import axios from 'axios';
 import HeaderComp from './Frame/HeaderComp/HeaderComp';
 import FreeWeekRotationComp from './Components/FreeWeekRotationComp/FreeWeekRotationComp';
 import SearchFieldComp from './Components/SearchFieldComp/SearchFieldComp';
+import MatchHistoryComp from './Components/MatchHistoryComp/MatchHistoryComp';
+import {getFullUrl} from './Modules/Helper';
 import "./App.css";
 
 class App extends React.Component {
 	state = {
 		champions: null,
```

Comment:
App module shouldn't have business-logic state. It only contains the application and the main router. move state and corresponding JSX to a different module